libs/ts/contracts/tasks/multichain-deploy.ts

```diff
@@ -261,19 +261,20 @@ const saveDeployment = async (
   configs: NetworkConfig[],
   chainsDeployment: DeploymentConfigV2,
 ) => {
-  const fileName = 'evm_contracts_deployment_v2';
-  const { decodeJSON, writeJSON } = selectDirectory(configDir);
-
-  const deploymentContent = await decodeJSON(
-    { name: fileName },
-    DeploymentConfigSchemaV2,
-  ).catch(() => ({}) as DeploymentConfigV2);
+  const { writeJSON } = selectDirectory(configDir);
```

Comment:
in [libs/ts/base-utils/src/env/constants.ts](https://github.com/blocksense-network/blocksense/blob/6a9debe431261b0c05988994f1f730dfc384aec4/libs/ts/base-utils/src/env/constants.ts#L14) you can add 
```diff
+  ['evm_contracts_deployment_v2']: join(
+    configDir,
+    'evm_contracts_deployment_v2',
+  ),
```

And then 
```diff
- const { writeJSON } = selectDirectory(configDir);
Comment
+  const { writeJSON } = selectDirectory(
+    configFiles.evm_contracts_deployment_v2,
+  );
```